client/src/app/project-registration/project-registration-form/project-registration-form.component.ts

```diff
@@ -170,6 +144,37 @@ export class ProjectRegistrationFormComponent implements OnInit {
     return false;
   }
 
+  private autofillProjectDetails(id, search) {
```

Comment:
it was changed slightly to use pipe + map instead of subscribe, in this commit: https://github.com/ebi-ait/ingest-ui/pull/86/commits/2ce67fb465fa96d6bd948f03d768ebe48db97e6f